listenbrainz/webserver/views/settings.py

```diff
@@ -274,32 +278,38 @@ def music_services_details():
     soundcloud_user = soundcloud_service.get_user(current_user.id)
     current_soundcloud_permissions = "listen" if soundcloud_user else "disable"
 
+    apple_service = AppleService()
+    apple_user = apple_service.get_user(current_user.id)
+    current_apple_permissions = "listen" if apple_user["refresh_token"] else "disable"
```

Comment:
`apple_user` can be empty in this case, which is throwing an error.

```suggestion
    current_apple_permissions = "listen" if apple_user and apple_user["refresh_token"] else "disable"
```